### ISSUE TITLE
Add a new issue/epic directly related to a pipeline

### ISSUE THREAD

**Issue by skalmus** [ZenHubIO/support#1111]
When adding a issue/epic, it would be much more productive if it was possible to add them direct from some specific pipeline. (Maybe as an item in pipeline options gear button)

**Comment by kendramoroz** [comment]
@skalmus great suggestion! Bumping over to discussion for further input/brainstorming here. 

When creating a new issue you're able to modify the pipeline it belongs to prior to submission, but the default will still remain as the far left pipeline. Improving this default and the workflow you mentioned is on our radar so we'll continue to provide updates here as they become available. 

Keep the feedback coming and let me know if you have any questions.

**Comment by adamreisnz** [comment]
I would like to add my vote here. We're using Trello in parallel to Zenhub for non-technical issues and tasks, and I find that I am missing the ability to create an issue directly in a certain pipeline, like you can do in Trello:

![image](https://user-images.githubusercontent.com/490562/32673326-a363df2e-c6b3-11e7-9372-70c254fa709e.png)

![image](https://user-images.githubusercontent.com/490562/32673336-b069956a-c6b3-11e7-83f3-9d36f96b547a.png)

This really saves a lot of time, because first of all you don't need to wait on a modal to show up and load, it also means you don't need to click pipeline and move the issue across. I rarely create issues that have to go to our first (backlog) pipeline, so this could be a big timesaver for us.

**Comment by bhack** [comment]
What is the concept of pipeline related to Epics? I never understand how to handle epics movement in pipelines.

**Comment by kendramoroz** [comment]
Hey @bhack great question here! We've seen a lot of teams create an Epic pipeline to store Epics, as they are less likely to move throughout the Board, as the Issues assigned to them should be the ones that flow through your workflow. The Epic pipeline is a great 'home' so the Epics can be collapsed in the pipeline for those that don't surface them to check on progress day-to-day.

As you mentioned, they're less likely to have movement compared to their Issues. This is something we have on our radar to address: providing a proper Epic management page to help with this exact workflow!

Let me know how the 'Epic' pipeline suggestion goes for your team in the meantime :).
